common/bootstrap/templates/prometheus.yaml

```diff
@@ -66,54 +66,8 @@ spec:
               - /etc/config/rules
               - /etc/config/alerts
         
+            {{- with $.Values.prometheus.server.extraScrapeConfigs }}
```

Comment:
i think this line should be directly above `toYaml` function
the `scrape_config` section would be removed if there is no `extraScrapeConfigs`

Reply:
I thought it will always scrape Kubernetes nodes with cAdvisor when running Prometheus. I thought it is better not to run Prometheus there is nothing to scrape and store. But now, I'll move those rules to 9c-main cluster.

Reply:
Fixed it, thank you!

https://github.com/planetarium/9c-infra/compare/75acad61d45d043d46c69d9bdba3826043798c24..89b58e687b48fdb78c25339519f108cc23456bd0